service/models.py

```diff
@@ -95,6 +95,16 @@ def create(self):
 
         if document.exists():
             self.id = document['_id']
+    
+    def update(self):
```

Comment:
Have we added test cases for update method?

Reply:
added the test function.